kymatio/scattering1d/conv_primitive.py

```diff
@@ -0,0 +1,27 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+class Conv1dFFTPrimative(nn.Module):
+    def __init__(self, backend):
+        super(Conv1dFFTPrimative, self).__init__()
+        self.name = 'torch'
+        self.backend = backend
+
+    def preprocess_signal(self, x, pad_left=None, pad_right=None):
+        if pad_left is not None:
```

Comment:
I could fuzz attack this by putting `pad_left=5, pad_right=None` and probably cause an error.
And with `pad_left=None, pad_right=5` an inaction.